### PR TITLE
Hw7

### PR DESCRIPTION
**Pull request by Stalexus**

### CODE REVIEW

File: style7.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    cursor: pointer;
+    border: none;
+    font-family:'Open Sans', sans-serif ;
+}
+.footer__contact {
+    height: 164px;
+    display: flex;
+    align-items: center;
+    justify-items: center;
+    background-color: black;
```
Comment (KseniaKotova) [inline style7.css]: It'll better to write color like #000000

File: index7.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <link rel="stylesheet" href="style7.css">
+    <title>hw7</title>
+</head>
+<body>
+    <header>
+        <h2>WHAT OUR CUSTOMERS ARE SAYING</h2>
+        <hr>
+        <h5>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+    </header>
+    <main class="mid">
```
Comment (egkirpichev) [inline index7.html]: I would personally name classes based on the content. For example, this section contains product reviews, therefore I would name it "reviews"

File: index7.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    <footer>
+            <div class="footer">
+                <div class="footer__contact">
+                    <a href="#"><img src="img/facebook_icon.png" alt="Facebook"></a>
+                    <a href="#"><img src="img/twitter_icon.png" alt="Twitter"></a>
+                    <a href="#"><img src="img/linkedin_icon.png" alt="Linkedin"></a>
+                    <a href="#"><img src="img/pinterest_icon.png" alt="Pinterest"></a>
+                    <a href="#"><img src="img/instagram_icon.png" alt="Instagram"></a>
+                    <a href="#"><img src="img/github_icon.png" alt="GitHub"></a>
+                    <a href="#"><img src="img/rss_icon.png" alt="rss"></a>
```
Comment (egkirpichev) [inline index7.html:82]: You've missed the credentials at the bottom of the page

File: style7.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
+body {
+    max-width: 1510px;
+    font-family:'Raleway', sans-serif ;
```
Comment (egkirpichev) [inline style7.css:3]: Your fonts are not plugged in anywhere. Therefore everything is rendered in Areal

File: style7.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    border: 2px solid #FFFFFF;
+    font-size: 18px;
+}
+.mid__four {
+    height: 170px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.mid__five {
```
Comment (egkirpichev) [inline style7.css]: This section's positioning is a bit off

File: style7.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    margin-left: -100px;
+}
+.mid__three {
+    height: 440px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    color: #FFF;
+    background-image: url(img/image.png);
```
Comment (egkirpichev) [inline style7.css]: Background picture was not dimmed. As a result this section appears too bright. Here is a few ways of how to make it look darker https://www.delftstack.com/howto/css/darken-background-image-css/

**Review by egkirpichev — APPROVED** [review]

**Review by KseniaKotova — APPROVED** [review]

File: style7.css [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+.mid__one__h:after {
+    content: "";
+    position: absolute;
+    border: 20px solid transparent; border-top: 30px solid #E2E2E2;
+    margin-top: 180px;
+    margin-right: 200px;
+}
+img {
+    position: absolute;
+    margin-left: -100px;
```
Comment (shuliakleonid) [inline style7.css]: position img dont work without top and ect.

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: style7.css (hunk 7 above)
Comment (Stalexus) [reply style7.css]: done

File: index7.html [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+</head>
+<body>
+    <header>
+        <h2>WHAT OUR CUSTOMERS ARE SAYING</h2>
+        <hr>
+        <h5>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+    </header>
+    <main class="main">
+        <div class="main__txt">
+            <h5 class="main__txt_h">Lorem ipsum dolor sit amet,
```
Comment (shuliakleonid) [inline index7.html:19]: Block Element Modify 
class main__txt_h is not correct 
you should named like main__title

File: index7.html [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+            </div>
+            <div>
+                <img class="photo" src="img/Ellipse 7 (2).png" alt="">
+                <h4>Cooper Dorwart</h4>
+                <h5>Manager</h5>
+            </div>
+        </div>
+        <div class="main__picture">
+            <h2>STYLISH AXURE DESIGN</h2>
+            <hr class="main__picture_hr">
```
Comment (shuliakleonid) [inline index7.html:54]: note correct naming

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**